web/src/markdown.js

```diff
@@ -675,14 +675,15 @@ export function initialize(helper_config) {
     web_app_helpers = helper_config;
 }
 
-export function apply_markdown(message) {
+export function get_parsed_data(raw_content) {
```

Comment:
I don't like this name; `apply_markdown` was fine, probably `markdown.render` would be better if we're renaming it. The point is we're not getting data that already exists -- we're actually doing the work of rendering the raw Markdown/text input into HTML output, and also returning a few extra fields about what we learned when doing so.

Reply:
Ah okay, sure!